src/licenses.rs

```diff
@@ -297,48 +300,71 @@ pub fn fetch_license_for_python_dependency(name: &str, version: &str) -> String
 
 /// Fetch the license for a Go dependency from the Go Package Index (pkg.go.dev)
 pub fn fetch_license_for_go_dependency(name: &str, _version: &str) -> String {
```

Comment:
```suggestion
pub fn fetch_license_for_go_dependency(name: Impl Into<String>, _version: Impl Into<String>) -> String {

let name = name.into()
let _version = _version.into()
```

Reply:
this'll cause other issues. will revisit later.